bin/reth/src/node/mod.rs

```diff
@@ -295,7 +295,9 @@ impl<Ext: RethCliExt> NodeCommand<Ext> {
         let head = self.lookup_head(Arc::clone(&db)).wrap_err("the head block is missing")?;
 
         // setup the blockchain provider
-        let factory = ProviderFactory::new(Arc::clone(&db), Arc::clone(&self.chain));
+        let (highest_snapshots_tx, highest_snapshots_rx) = watch::channel(None);
```

Comment:
where does the sender go here, looks unused now.

I think we should consider hiding this channel and move instantiation to the Snapshotter instead, it can keep both halfs and return a new receiver on demand

Reply:
https://github.com/paradigmxyz/reth/blob/113ef23de1c09c008ced21718c514369d5e0a17a/bin/reth/src/node/mod.rs#L469-L474

Reply:
i think it still makes sense to move it inside